spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/pom.xml

```diff
@@ -45,6 +45,18 @@
 			<scope>test</scope>
 		</dependency>
 
-	</dependencies>
+		<dependency>
+			<groupId>com.google.cloud.functions.invoker</groupId>
+			<artifactId>java-function-invoker</artifactId>
+			<version>1.0.0-alpha-2-rc3</version>
```

Comment:
make it a property or use `google.cloud.functions.version` one

Reply:
done.